content/docs/concepts/architecture/index.md

```diff
@@ -54,25 +55,26 @@ api, bridge, or resource-service.
 
 ### api-service
 
-The [Keptn API](../../0.18.x/reference/api/) provides a REST API
+The [Keptn API](../../0.19.x/reference/api/) provides a REST API
 that allows you to communicate with Keptn.
 It provides endpoints to authenticate, get metadata about the Keptn installation within the cluster,
-forward CloudEvents to the NATS cluster, and triggering evaluations for a service.
-
-The documentation shows you how to access the API and to explore the available endpoints.
+forward [CloudEvents](../../0.19.x/reference/miscellaneous/events)
+to the NATS cluster, and trigger evaluations for a service.
 
 ### mongodb-service
 
 The *mongodb-datastore* stores event data in a MongoDB deployed in your Keptn namespace.
```

Comment:
In addition to having the mongodb deployed in the Keptn namespace (as it is by default), there is also a possibility of pointing to an externally hosted mongodb instance via this helm value: https://github.com/keptn/keptn/blob/539339ef3da8e55d7968852a88ed01e0088f5871/installer/manifests/keptn/values.yaml#L47

Reply:
done